File: PhonoArk/PhonoArk/Models/AppSettings.cs
namespace PhonoArk.Models;

public class AppSettings
{
    public int Id { get; set; }
    public Accent DefaultAccent { get; set; } = Accent.GenAm;
    public double Volume { get; set; } = 0.8;
    public int ExamQuestionCount { get; set; } = 10;
    public bool DarkMode { get; set; } = false;
    public bool RemindersEnabled { get; set; } = false;
}

File: pureopus/app/src/main/java/com/phonoark/data/model/AppSettings.kt
package com.phonoark.data.model

data class AppSettings(
    val id: Long = 1,
    val defaultAccent: Accent = Accent.US_JENNY,
    val volume: Int = 80,
    val examQuestionCount: Int = 10,
    val darkMode: Boolean = false,
    val remindersEnabled: Boolean = false,
    val language: String = "en"
)
